Misc/NEWS.d/next/Library/2024-09-06-11-58-17.gh-issue-123742.jSBNUv.rst

```diff
@@ -0,0 +1,2 @@
+Fix :meth:`email.Message.get_payload` couldn't parse headers
+with extra spaces. Patch by RUANG.
```

Comment:
Please read again the section about how to write NEWS entries:

> Describe the user-visible effects of your change succinctly and accurately [...]

Now, what is the _user-visible effects_ of this PR?